CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -216,6 +212,109 @@ function PlayerHospital:dailyAdviceChecks()
   end
 end
 
+--! Give advice about the player's progress towards winning or losing the level
+function PlayerHospital:giveLevelProgressAdvice()
+  -- Starting position (measure progress from here towards win/lose)
+  local level_config, level, start = self.world.map.level_config, self.world.map.level_number, {}
+  if level_config.towns and level_config.towns[level] then
+    start["balance"] = level_config.towns[level].StartCash
+    start["reputation"] = level_config.towns[level].StartRep
+  elseif level_config.town then
+    start["balance"] = level_config.town.StartCash
+    start["reputation"] = level_config.town.StartRep
+  end
+  if not start["balance"] then start["balance"] = 0 end
```

Comment:
Confirm, but base_config indicates the default `StartCash` is 40000.
`StartRep` is also 500 in that config. 
So why do we need to declare a failure case value here for both lines?

Reply:
`StartRep` was only introduced in 0.65 -- probably why you have that failsafe then.